### ISSUE TITLE
Verilog: reset signal is inconsistent?

### ISSUE THREAD

**Issue by HazeSystem** [sehugg/8bitworkshop#132]
Using Ball Motion (absolute position) as an example, if one clicks reset over and over at the start of running the code while the ball is moving to the top-left of the screen, it seems to reset correctly. However, if you let the ball bounce so it's then traveling to the bottom-right and reset it, the ball moves back to the center of the screen but keeps the same direction from before the reset, traveling towards the bottom-right.

**Comment by sehugg** [comment]
You're right, the velocity X/Y values don't respond to the reset signal. I fixed the module in the dev build.

See also: https://github.com/sehugg/8bitworkshop/issues/23
